crates/sui-core/src/consensus_handler.rs

```diff
@@ -191,14 +194,21 @@ impl<T: ParentSync + Send + Sync> ExecutionState for ConsensusHandler<T> {
             .consensus_committed_subdags
             .with_label_values(&[&leader_author.to_string()])
             .inc();
-        for (cert, batches) in consensus_output.batches {
+        for (cert, batches) in consensus_output
+            .sub_dag
+            .certificates
+            .iter()
+            .zip(consensus_output.batches.iter())
```

Comment:
Is our intention to eventually remove the `certificate` from the `batches` vector? Otherwise why this change?